keyboards/yd60mq/yd60mq.c

```diff
@@ -1,12 +1,14 @@
 #include "yd60mq.h"
 
-void led_set_kb(uint8_t usb_led) {
-    if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
-        setPinOutput(F4);
```

Comment:
Do we need an init function to set F4 as output?

Reply:
I felt like something wasn't quite right here.